src/views/product-landing/components/overview-cta/index.tsx

```diff
@@ -39,6 +41,7 @@ function OverviewCta({
 			<div className={s.imagePart}>
 				<img src={image} alt="" />
 			</div>
+			{fullWidthBody ? <p className={s.body}>{fullWidthBody}</p> : null}
```

Comment:
I added support for `fullWidthBody` prop on this component. The schema is set up to only accept either a cta or this fullWidthBody prop, not both.

Reply:
I had the same thought re: everything being blocks but didn't want to take on the refactor. I'll check out making this a generic paragraph!

Reply:
Updated with your suggestions @zchsh!